Client/game_sa/CRadarSA.cpp

```diff
@@ -175,16 +175,14 @@ VOID CRadarSA::ClearMarkerForEntity(CPed* ped)
     }
 }
 
-struct CRect
-{
-    float fX1, fY1, fX2, fY2;
-};
-
 VOID CRadarSA::DrawAreaOnRadar(float fX1, float fY1, float fX2, float fY2, const SColor color)
 {
     // Convert color to required abgr at the last moment
     unsigned long abgr = color.A << 24 | color.B << 16 | color.G << 8 | color.R;
-    CRect         myRect;
+    struct
+    {
+        float fX1, fY1, fX2, fY2;
+    } myRect;
```

Comment:
Side question: how come this is a custom struct and not a CRect? Is a `CRect` just too heavyweight?

Reply:
Ah correct, this should be CRect. MTA has its own version of CRect, it's called CRect2D. I confused this with CRect2D.

Reply:
Might be useful to include a comment on the CRect class that explains when and why it should be used